api/rest/triple.yaml

```diff
@@ -0,0 +1,109 @@
+---
+swagger: '2.0'
+info:
+  title: Data Commons API
+  description: Given a list of nodes, return triples which are associated with the specified node(s).
+  version: 1.0.0
+host: api.datacommons.org
+schemes:
+- https
+paths:
+  "/node/triples":
+    get:
+      summary: Given a list of nodes, return triples which are associated with the
+        specified node(s).
+      description: Intended for users willing to modify the URLs called from their
```

Comment:
I'm not sure if we should focus all our REST descriptions with cURL - there are many ways to access REST APIs. Along those lines, that is one reason for using GET, though it is also useful for shorter queries. POST was added for longer queries that won't get bound by the URL length limit.